Sources/StreamChat/APIClient/Endpoints/PollsEndpoints.swift

```diff
@@ -0,0 +1,170 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+extension Endpoint {
+    static func createPoll(createPollRequest: CreatePollRequestBody) -> Endpoint<PollPayload> {
+        .init(
+            path: .polls,
+            method: .post,
+            queryItems: nil,
+            requiresConnectionId: false,
+            requiresToken: true,
+            body: createPollRequest
+        )
+    }
+    
+    static func updatePoll(updatePollRequest: UpdatePollRequestBody) -> Endpoint<PollPayload> {
```

Comment:
This would be then:
`static func updatePoll(updatePollRequest: UpdatePollRequestBody) -> Endpoint<PollPayloadResponse> {`
or 
`static func updatePoll(updatePollRequest: UpdatePollRequestBody) -> Endpoint<PollResponse> {`

I'm fine with both `PollPayloadResponse` or `PollResponse`. What is important is that it has "Response" post-fix